editor/import/resource_importer_wav.cpp

```diff
@@ -35,6 +35,15 @@
 #include "core/io/resource_saver.h"
 #include "scene/resources/audio_stream_wav.h"
 
+#define DRWAV_IMPLEMENTATION
+#define DR_WAV_NO_STDIO
+#define DR_WAV_LIBSNDFILE_COMPAT
```

Comment:
Despite not using `libsndfile`, the way Godot converts streams from/to f32 follows its calculations, therefore not defining this leads to some test errors.

Reply:
Horray